OTAnalytics/plugin_ui/nicegui_gui/dialogs/edit_section_dialog.py

```diff
@@ -45,11 +45,8 @@ def __init__(
 
     def build_content(self) -> None:
         self._name.build()
-        try:
-            self._name.element.props(f"test-id={MARKER_NAME}")
-        except Exception:
-            # .props might not be available in all NiceGUI versions; ignore gracefully
-            pass
+        self._name.element.props(f"test-id={MARKER_NAME}")
```

Comment:
This will already be done in FormField#_apply_marker. Please remove the code and check the remaining code about similar code.